package.json

```diff
@@ -63,7 +64,7 @@
     "deps:extra": "dependency-check package.json --extra --no-dev --ignore",
     "audit": "nsp check",
     "precheck": "npm test",
-    "check": "npm run audit && npm run deps && npm outdated --depth 0",
+    "check": "npm run audit && npm run deps",
```

Comment:
Removing `npm outdated` since
* we have greenkeeper
* it exit code is 1 now on npm v4 :(